src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task{
+    private LocalDateTime at;
```

Comment:
I know its easy to tell from the context, but maybe the variable name can be a little more descriptive? I noticed this in `Deadline.java` too 👍